rest/data/project_test.go

```diff
@@ -141,9 +142,14 @@ func TestProjectConnectorGetSuite(t *testing.T) {
 				EventType:    event.EventTypeProjectModified,
 				ResourceId:   projectId,
 				Data: &model.ProjectChangeEvent{
-					User:   username,
-					Before: before,
-					After:  after,
+					User: username,
+					Before: model.ProjectSettingsEvent{
```

Comment:
One thing I would also like to check in a test before going forwards is if we have a ProjectChangeEvent that is in the old format (i.e. Before is a ProjectSettings struct) does it still unmarshal correctly when retrieving from the database?